### PR TITLE
Skills component

### PR DESCRIPTION
**Pull request by Nadeen123**
related to #8 
created Skills component with filter
created filter styled component
created SkillsStyled component
data file

### CODE REVIEW

File: client/src/components/Skills/FilterStyledComponent.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components"
+
+const DivStyled = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 5px;
+  margin-bottom: 10px;
+  margin: auto;
```
Comment (FarahZaqout) [inline client/src/components/Skills/FilterStyledComponent.js]: collapse the margins into one line.
```suggestion
  margin: 5px auto 10px;
```

File: client/src/components/Skills/FilterStyledComponent.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+`
+const FilterStyled = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border: black;
+`
+
+const FilterButton = styled.button`
+  background: ${({ background }) => background || "white"};
```
Comment (FarahZaqout) [inline client/src/components/Skills/FilterStyledComponent.js]: use ```#fff``` instead of ```white```

File: client/src/components/Skills/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            <FilterButton value="everyday" onClick={this.onClick}>
+              Everyday
+            </FilterButton>
+            <FilterButton value="superpower" onClick={this.onClick}>
+              Superpower
+            </FilterButton>
+          </FilterStyled>
+        </DivStyled>
+
+        {this.state.FilteredData.map(e => {
```
Comment (FarahZaqout) [inline client/src/components/Skills/index.js]: we should always check the existence of the list we are mapping.
```suggestion
        {this.state.FilteredData && this.state.FilteredData.map(e => {
```

File: client/src/components/Skills/index.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+          console.log(e)
```
Comment (FarahZaqout) [inline client/src/components/Skills/index.js]: remove console.

**Comment by Nadeen123** [comment]
@FarahZaqout Resolved comments

**Pull request merged**